aws-ecr-login/README.md

```diff
@@ -0,0 +1,49 @@
+# AWS ECR Login
+
+This task retrieves an `authentication token` using the GetAuthorizationToken API 
+that you can use to authenticate to an `Amazon ECR registry`. You can pass the 
+authorization token to the login command of the container client of your preference,
+such as the Docker CLI. After you have authenticated to an Amazon ECR registry with 
+this authentication token, you can use the client to push and pull images from that 
+registry as long as your IAM principal has access to do so until the token expires. 
+
+*NOTE*: The authorization token is valid for 12 hours.
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/aws-ecr-login/aws-ecr-login.yaml
```

Comment:
- Can you please update the install path command as well like when all the files are moved to the task directory

   The install command path would look like

   kubectl apply -f 
   https://raw.githubusercontent.com/tektoncd/catalog/master/task/aws-ecr-login/0.1/aws-ecr-login.yaml

Reply:
Sure, will do the changes based on the re-org :)